src/test/java/org/elasticsearch/test/rest/ElasticsearchRestTestCase.java

```diff
@@ -46,22 +54,44 @@
 import org.junit.Test;
 
 import java.io.IOException;
-import java.nio.file.FileSystems;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 import java.nio.file.Path;
 import java.nio.file.PathMatcher;
-import java.nio.file.Paths;
-import java.util.*;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
 
 /**
  * Runs the clients test suite against an elasticsearch cluster.
  */
-//tests distribution disabled for now since it causes reporting problems,
-// due to the non unique suite name
-//@ReplicateOnEachVm
-@AbstractRandomizedTest.Rest
+@ElasticsearchRestTestCase.Rest
+@Slow
```

Comment:
oh this should totally be a `@Slow` test - it takes over a minute on my notebook even though it has lots of smallish tests.